assets/js/state/hosts.js

```diff
@@ -43,6 +43,14 @@ export const hostsListSlice = createSlice({
         return host;
       });
     },
+    updateSelectedChecks: (state, action) => {
+      state.hosts = state.hosts.map((host) => {
+        if (host.id === action.payload.targetID) {
```

Comment:
why `targetID` and not `hostID`?

Reply:
Because that is what we get from the action payload as it currently is. 

Can change as per this [comment](https://github.com/trento-project/web/pull/1618/files#r1261176463)